apps/challenges/serializers.py

```diff
@@ -63,11 +64,55 @@ class ChallengePhaseSplitSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ChallengePhaseSplit
-        fields = '__all__'
```

Comment:
Actually, this line is updated in the line just after it. So, there is no use of this line. That's why I removed it. Please check [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/serializers.py#L66).